.github/workflows/ci.yml

```diff
@@ -147,9 +165,6 @@ jobs:
             - uses: Swatinem/rust-cache@v2
             - name: Check
               run: cargo check --workspace --target wasm32-unknown-unknown
-            # TODO: [#2191](https://github.com/gakonst/ethers-rs/issues/2191)
```

Comment:
is this no longer a problem?

Reply:
hmm i forget why i removed that, it still is with the features